s3fs/core.py

```diff
@@ -71,6 +71,59 @@ def split_path(path):
              'authenticated-read'}
 
 
+def title_case(string):
+    """
+    TitleCases a given string.
+
+    Parameters
+    ----------
+    string : underscore seperated string
+    """
+    return ''.join([x.capitalize() for x in string.split('_')])
+
+
+class SSEParams(object):
```

Comment:
I think it is still helpful to provide some settings that users can discover via tab completion.